src/components/vm/disks/diskAdd.jsx

```diff
@@ -244,7 +240,7 @@ const AdditionalOptions = ({ cacheMode, device, idPrefix, onValueChanged, busTyp
                         aria-label={_("serial number")}
                         className="ct-monospace"
                         value={serial}
-                        onChange={(_, value) => setSerialHelper(value)} />
+                        onChange={(_, value) => onValueChanged("serial", value)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1303-20231109-142627-1ac1b404-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)